src/server/configs/root.config.ts

```diff
@@ -0,0 +1,30 @@
+import express from "express";
+import cors from "cors";
+import path from 'path';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export const envConfig = {
+    mongo_uri: process.env.MONGO_URI as string,
+    access_secret: process.env.ACCESS_SECRET as string,
+    port: process.env.PORT || 3000,
+
+}
+
+const __dirname = path.resolve();
+const clientPath = path.join(__dirname, '/dist/client');
+
+console.log(__dirname)
+
+export const app = express();
+
+app.use(cors());
+app.use(express.json())
+app.use(express.static(clientPath));
+
+
+app.all('*', function(req, res) {
```

Comment:
I WAS going to fight against you on this haha. My reason was that since all files in the server run at the same time, and as long as they are imported/exported, then they should behave as normal. BUUUT I am wrong and you are right because before I pushed, I forgot to test if i can sign in successfully and I get 404 error. 

Got so excited about this abstraction lol so my apologies! Should be resolved now... Going to merge